frontend/src/components/home/Home.scss

```diff
@@ -0,0 +1,4 @@
+.align-items-bottom {
+    display: flex;
+    align-items: flex-end;
+}
```

Comment:
Met dat er hier op het einde van de file geen newline is: heb je `editorconfig` plugin installed?